GSA-TTS/FAC
# Copy Updates Re: Our Lack of Email Notifications

# Copy Updates Re: Our Lack of Email Notifications

Issue: https://github.com/GSA-TTS/FAC/issues/3182

## Changes:
There have been copy changes in a few notable places:
1. The audit access form (step 3). 
2. The lock for certification page.
3. The auditor certification page (not auditee, since they're last). 

There are also some very minor padding changes.

The relevant issue also notes a change to the "collapsed Submission checklist", which is handled [here](https://github.com/GSA-TTS/FAC/pull/3197).

## How to test:
1. Switch to this branch and run normally.
2. Start a submission. Verify the text addition in the audit access form (right before the general info form).
3. Step through to certification. Verify the text addition in the lock page and auditor certification page.

## Screenshots:
Audit access:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/9124ff4a-074c-4936-a43e-6db027c3eae1)

Lock for certification:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/accb6ccf-503b-4b2b-b05c-a1f889716794)

Auditor certification:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/700b6b02-b082-4ed6-b71d-74014c1831ce)

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
